src/utils/IdealGraphVisualizer/ServerCompiler/src/com/sun/hotspot/igv/servercompiler/filters/onlyControlFlow.filter

```diff
@@ -20,5 +20,5 @@ f.addRule(
     ), false
   )
 );
-f.addRule(new RemoveFilter.RemoveRule(new MatcherSelector(new Properties.RegexpPropertyMatcher("name", "Phi|Store."))));
+f.addRule(new RemoveFilter.RemoveRule(new MatcherSelector(new Properties.RegexpPropertyMatcher("name", "Phi|CreateEx|Cast.*|Load.|Store."))));
```

Comment:
I tried the filter out on a graph and noticed that there are still some data and memory nodes shown. How about utilizing the newly added categories? I played around a bit and this seems to work quite nicely. What do you think?

```
var f = new RemoveFilter("Show only control flow");
f.addRule(
  new RemoveFilter.RemoveRule(
    new OrSelector(
      new InvertSelector(
        new MatcherSelector(
          new Properties.RegexpPropertyMatcher("category", "control|mixed|other")
        )
      ),
      new MatcherSelector(
        new Properties.StringPropertyMatcher("type", "abIO")
      )
    ), false
  )
);
f.addRule(new RemoveFilter.RemoveRule(new MatcherSelector(new Properties.RegexpPropertyMatcher("name", "Phi|Root|Con"))));
f.apply(graph);
```

Reply:
Thanks for the suggestion, I adopted it with some variations to avoid having to explicitly list pinned nodes like "Phi".

Reply:
That's even better! Thanks for adopting that suggestion.